.env.example

```
@@ -0,0 +1,5 @@
+# Database
+DATABASE_URL="postgresql://dependenga:dependenga@localhost:5432/dependenga"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)